core/modules/savers/gitlab.js

```diff
@@ -0,0 +1,122 @@
+/*\
+title: $:/core/modules/savers/gitlab.js
+type: application/javascript
+module-type: saver
+
+Saves wiki by pushing a commit to the GitLab REST API
+
+\*/
+(function(){
+
+	/*jslint node: true, browser: true */
+	/*global $tw: true */
```

Comment:
the `(function(){` ... doesn't count for indentation. So you'll need to go 1 level in.